feat: support connectTimeout

maoweipeng opened the pull request.

maoweipeng committed:
feat: support connectTimeout

Rapsssito commented:
Hi @maoweipeng,

Thanks for the PR, could you please add this change to the documentation?

maoweipeng committed:
doc: supplement the connectTimeout documentation

Rapsssito committed:
Improve connectTimeout description formatting

Removed HTML line breaks from the connectTimeout description for better readability.
